### ISSUE TITLE
Cannot click to close some application folder in GNOME 46

### ISSUE THREAD

**Issue by joezhouchenye** [l3nn4rt/click-to-close-overview#6]
![image](https://github.com/l3nn4rt/click-to-close-overview/assets/59191277/d8076ff4-3781-436a-825e-1ca6635607b6)

Just upgraded from ubuntu 23.10 to 24.04. Don't know what is affecting the behavior. For example, I cannot click to close "系统" if I open it. However, I can click to close "工具". To solve this, I have to drag all apps out and recreate the application folder.

Also, "预览" used to work, but after the recreation of “工具”, it doesn't work now.

**Comment by joezhouchenye** [comment]
Maybe a problem with Ubuntu Dock Extension. I switched to Dash to Dock and the problem is gone.

**Comment by l3nn4rt** [comment]
Sorry, I've been AFK for a while.
Is this solved?

**Comment by joezhouchenye** [comment]
No, the problem is still there. It's not the Dock extension's problem.

A reboot can solve this. However, the unexpected behavior happens again after opening some apps in the folder (not everytime).

I don't know what triggers this behavior. I've attached a video below.

[Peek 2024-10-08 09-37.webm](https://github.com/user-attachments/assets/7d114959-c834-4a0a-96f5-80444635bb59)

**Comment by l3nn4rt** [comment]
I think the problem you're facing is caused by some extension conflict.
In order to identify which one is conflicting with click-to-close-overview, please, try the following:

1. disable all extensions but this one
2. restart the shell (logout & login)
3. enable another extension
4. check if the error happens
5. if not, got to step 3

Please, let me know which is the conflicting extension, so I can try to reproduce it.

Bonus: looks like you're using Blur My Shell. I'm using it too without any problem, so it should not be the one

**Comment by l3nn4rt** [comment]
Closing
